zgrab2_schemas/zgrab2/ipp.py

```diff
@@ -147,15 +147,52 @@
     "request": http_request_full
 })
 
+# TODO: Determine whether value-tag types with same underlying form should have a different name in this mapping
+ipp_attribute_value = SubRecord({
+    "raw": Binary(),
+    "integer": Signed32BitInteger(),
+    "boolean": Boolean(),
+    "enum": String(),
+    # TODO: Determine appropriate type for octetString w/o specified format
```

Comment:
`Binary()` is the proper type for `octetString`; if there is another value encoded inside the octet string, and we need to access the fields of that value, then that would need special handling.